Library/Homebrew/language/python.rb

```diff
@@ -228,9 +231,33 @@ def virtualenv_install_with_resources(using: nil, system_site_packages: true, wi
           python = T.must(wanted.first)
           python = "python3" if python == "python"
         end
+
+        venv_resources = if without.blank? && start_with.blank? && end_with.blank?
```

Comment:
```suggestion
        venv_resources = if without.nil? && start_with.nil? && end_with.nil?
```

Nit: IMO we should be stricter with the inputs here unless there is a case where we'd expect someone to explicitly provide an empty string or array and expect it to get ignored.